sources/osgGA/OrbitManipulatorHammerController.js

```diff
@@ -56,29 +60,36 @@ OrbitManipulatorHammerController.prototype = {
         this._cbPinchInOut = this.pinchInOut.bind(this);
 
         hammer.on('panstart ', this._cbPanStart);
+        hammer.on('pinchstart', this._cbPinchStart);
         hammer.on('panmove', this._cbPanMove);
         hammer.on('panend', this._cbPanEnd);
-        hammer.on('pinchstart', this._cbPinchStart);
         hammer.on('pinchend', this._cbPinchEnd);
         hammer.on('pinchin pinchout', this._cbPinchInOut);
     },
 
+    setEventProxy: function(hammer) {
+        if (!hammer || hammer === this._eventProxy) {
+            return;
+        }
+        this._eventProxy = hammer;
+        if (this._manipulator) this.setListeners();
+    },
+
     _computeTouches: function(event) {
+        // should not happen as we test mouse each time ?
         if (event.pointers !== undefined) return event.pointers.length;
+        console.log('compute touches mouse');
```

Comment:
never ever got into this log...

Reply:
hammer was also used on touchscreen not only mobile, it's possible it's an old stuff from the time touch was touch api was not stable.
If the code is useless, then remove it, be sure to test it on different device